src/Symfony/Component/Finder/Finder.php

```diff
@@ -397,13 +397,17 @@ public function sort(\Closure $closure)
      *
      * This can be slow as all the matching files and directories must be retrieved for comparison.
      *
+     * @param bool $naturalOrdering Whether to use natural ordering or not
+     *
      * @return $this
      *
      * @see SortableIterator
      */
-    public function sortByName()
+    public function sortByName($naturalOrdering = false)
```

Comment:
for BC concerns, this should be handled using `func_num_args()/func_get_arg(0)` (adding a new argument breaks child classes). 
The signature should then be written as `sortByName(/* bool $useNaturalOrdering = false */)`

Reply:
Most of the times ["sort" and "order" are interchangeable](https://ell.stackexchange.com/questions/127765/are-sort-and-order-interchangeable). However, given that our methos is called `sortByName`, we use a `$this->sort` property and PHP calls this "natsort" for "natural sort" ... could we please rename `$useNaturalOrdering` to `$useNaturalSort` ?